conda-forge.yml

```diff
@@ -1,5 +1,3 @@
-bot:
-  automerge: true
```

Comment:
@mikofski, I don't think the bot automerge is safe for this feedstock.  It's been working okay but the dependencies have changed a lot since that was turned on and it's the feedstock hasn't been updated accordingly.  I'm willing to maintain if you don't have time.